### DISCUSSION TITLE
[Feature] Optionally prevent photos reuploads after deletion

### DISCUSSION THREAD

**Discussion by alassagne-sl** [immich-app/immich#5935] [Feature Request]
### The feature

When I delete a photo from the server, it gets uploaded again by my phone. From what I can read, it’s working as designed, backing up every photo that is not on the server.

I’m using Immich not only for backup but also as a « main » library, so I delete duplicates and bad pictures from the server. For this usage, the current behavior could be improve. 

Feature example:
- an argument in the asset deletion request to choose (or not) the behavior
- a blacklist with these deleted photos
- A api route to clear the blacklist
- An option in the mobile app to force the reupload of an image 
- An option in the web app to delete an asset and « blacklist » it
- An option in the web app to force an upload if it was detected as blacklisted
- An option in the CLI to force reupload of blacklisted assets
(Maybe everything is not mandatory in a first version, I ordered by importance)

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by samholton** [comment]
Probably covered by https://github.com/immich-app/immich/discussions/3594

**Reply by antlassagne** [reply]
Indeed

**Reply by cleanboy** [reply]
No, I would disagree with that. The linked issue is for a feature that syncs the server with the phone in that when you delete from the server, it deletes from the phone. I want to delete from the server and it stays deleted on the server but can remain on the phone if I choose

**Comment by jazmit** [comment]
The best solution for this is #4282

**Reply by antlassagne** [reply]
Indeed, at this point this is a duplicate
